easybuild/easyconfigs/g/gettext/gettext-0.19.8.1-GCCcore-7.2.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'gettext'
+version = '0.19.8.1'
+
+homepage = 'http://www.gnu.org/software/gettext/'
+description = """GNU 'gettext' is an important step for the GNU Translation Project, as it is an asset on which we may
+build many other steps. This package offers to programmers, translators, and even users, a well integrated set of tools
+and documentation"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['ff942af0e438ced4a8b0ea4b0b6e0d6d657157c5e2364de57baa279c1c125c43']
+
+dependencies = [
+    ('libxml2', '2.9.8'),
+    ('ncurses', '6.1'),
+]
+
+# use same binutils version that was used when building GCCcore toolchain
+builddependencies = [('binutils', '2.29')]
+
+configopts = '--without-emacs --with-libxml2-prefix=$EBROOTLIBXML2'
+
+moduleclass = 'vis'
```

Comment:
@rubendibattista please fix according to the changes in #5991 (wrong `moduleclass` + missing custom `sanity_check_paths`)